src/client/components/PasswordInput.stories.tsx

```diff
@@ -1,23 +1,38 @@
 import React from 'react';
 import { Meta } from '@storybook/react';
 
-import { PasswordInput } from './PasswordInput';
+import { PasswordInput, PasswordInputProps } from './PasswordInput';
 
 export default {
   title: 'Components/PasswordInput',
   component: PasswordInput,
-} as Meta;
+} as Meta<PasswordInputProps>;
 
-export const Default = () => <PasswordInput label="Password" />;
+export const Default = (props: PasswordInputProps) => (
+  <PasswordInput {...props} label="Password" />
+);
 Default.storyName = 'default';
 
-export const WithError = () => <PasswordInput label="Password" error="Error" />;
+export const WithError = (props: PasswordInputProps) => (
+  <PasswordInput {...props} label="Password" error="Error" />
+);
 WithError.storyName = 'with error';
 
-export const WithSupporting = () => (
+export const WithSupporting = (props: PasswordInputProps) => (
   <PasswordInput
+    {...props}
     label="New password"
     supporting="Must be between 8 and 72 characters"
   />
 );
 WithSupporting.storyName = 'with supporting text';
+
+export const WithoutEye = (props: PasswordInputProps) => (
+  <PasswordInput
+    {...props}
+    label="New password"
+    supporting="Must be between 8 and 72 characters"
+    displayEye={false}
+  />
+);
```

Comment:
The spread props might need to come after the other props, if we want people viewing the story to be able to override all values.

Reply:
good spot, will change